features/machine/machine.feature

```diff
@@ -141,4 +141,39 @@ Feature: Machine features testing
     Given I scale the machineset to +2
     Then the machineset should have expected number of running machines
 
+  # @author miyadav@redhat.com
+  # @case_id OCP-24363
+  @admin
+  @destructive
+  Scenario: [MAO] Reconciling machine taints with nodes
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+
+    And I wait for the steps to pass:
+    """
+    Given I clone a machineset named "machineset-24363"
+    """
+    Then I store the last provisioned machine in the :machine clipboard
+    And evaluation of `machine(cb.machine).node_name` is stored in the :noderef_name clipboard
+
+    Given I saved following keys to list in :taintsid clipboard:
+     | {"spec":{"taints": [{"effect": "NoExecute","key": "role","value": "master"}]}} | |
+     | {"spec":{"taints": [{"effect": "NoSchedule","key": "role","value": "master"}]}}| |
```

Comment:
Please remove the redundant `|` in the able.

Reply:
I get this error if I do that -  The table must have exactly 2 columns (RuntimeError)